packages/jest-util/src/install_common_globals.js

```diff
@@ -13,10 +13,26 @@ import type {Global} from 'types/Global';
 import createProcesObject from './create_process_object';
 import deepCyclicCopy from './deep_cyclic_copy';
 
+const DTRACE = [
+  'NET_SERVER_CONNECTION',
+  'NET_STREAM_END',
+  'HTTP_SERVER_REQUEST',
+  'HTTP_SERVER_RESPONSE',
+  'HTTP_CLIENT_REQUEST',
+  'HTTP_CLIENT_RESPONSE',
+];
+
 export default function(globalObject: Global, globals: ConfigGlobals) {
   globalObject.process = createProcesObject();
 
   // Forward some APIs.
+  DTRACE.forEach(dtrace => {
+    globalObject['DTRACE_' + dtrace] = function() {
```

Comment:
can you use `...args` and arrow functions instead? arguments is bad.

Reply:
I need a proper function so that `this` can be captured. Using `...args` sounds unnecessary to me, since no processing as `Array` is needed.

Reply:
You are right about `this` but `...args` should be faster than arguments at this time.